data/styles/outdoors/include/Icons.mapcss

```diff
@@ -256,6 +256,8 @@ node|z16-[tourism=viewpoint],
 
 
 /* 5.5 Others */
+node|z18-[man_made=cairn],
+{text: name;text-color: @poi_label;text-offset: 1;font-size: 10;}
```

Comment:
I think just put both icon and caption declaration in one line, atm you duplicate some params like `font-size`.
Also in outdoors we use slightly bigger font sizes, for this zoom it should be `12`.

Reply:
fixed